runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/External.java

```diff
@@ -53,6 +55,7 @@
  * service. Note that this is a low-level API and mainly for internal use. A user may want to use
  * high-level wrapper classes rather than this one.
  */
+@Experimental(Kind.PORTABILITY)
```

Comment:
remove since this is runners core?

Reply:
I prefer to do an exception for this one because of https://issues.apache.org/jira/browse/BEAM-8546. This class is mislocated but nobody has cared about putting it on `sdks/java/core` where it belongs so this is to prevent than once it is moved there the 'experimental' annotation gets forgotten.

Reply:
sgtm